automatic/chromium/tools/chocolateyInstall.ps1

```diff
@@ -14,7 +13,7 @@ $validExitCodes = @(0)
   if (Test-Path $Chromium) {
     $silentArgs = ''
   } else {
-    $silentArgs = '--system-level --do-not-launch-chrome'
+    $silentArgs = '--system-level'
   }
   
-    Install-ChocolateyPackage $packageName $fileType $silentArgs $url $validExitCodes $checksum $checksumType
+    Install-ChocolateyPackage $packageName $fileType $silentArgs $url $checksum $checksumType

```

Comment:
This change won't resolve the problem. Now it's going to the checksum is url64bit